README.md

```diff
@@ -78,9 +55,7 @@ from featureflags.util import log
 
 def main():
     # Create a Feature Flag Client
-    client = CfClient("c9b3f14f-6336-4d23-83b4-73f29d1ebeeb",
-                      with_base_url("https://config.ff.harness.io/api/1.0"),
-                      with_events_url("https://events.ff.harness.io/api/1.0"))
+    client = CfClient("c9b3f14f-6336-4d23-83b4-73f29d1ebeeb")
```

Comment:
should we move the sdk key to it's own variable?

Reply:
yes good shout